.github/workflows/cron-streams-active-cleanup.yaml

```diff
@@ -12,6 +12,9 @@ jobs:
           - name: "staging"
             hostname: "livepeer.monster"
             api-token-secret: LP_STAGING_API_ADMIN_TOKEN
+          - name: production
```

Comment:
Don't we want to run stream clean-up as Kubernetes CronJobs rather than GH Actions? I think it should be simpler to monitor with our existing infra.

Reply:
Yeah agreed! I don't think it's in scope of this PR though as explained here: v